components/description_list/description_list.vue

```diff
@@ -0,0 +1,122 @@
+<!-- eslint-disable vue/require-v-for-key -->
+<template>
+  <dl :class="['dt-description-list', getDirectionClass, getGapClass]">
+    <template
+      v-for="item in items"
+    >
+      <dt
+        :class="['dt-description-list__term', termClass]"
+      >
+        {{ item.term }}
+      </dt>
+      <dd
+        :class="['dt-description-list__description', descriptionClass]"
+      >
+        {{ item.description }}
+      </dd>
+    </template>
+  </dl>
+</template>
```

Comment:
Ok, my 2 cents.
Remove `<!-- eslint-disable vue/require-v-for-key -->`

and update

```
<dt
        :key="`dt-${item.term}`"
        :class="['dt-description-list__term', termClass]"
      >
```

and

```
<dd
        :key="`dd-${item.term}`"
        :class="['dt-description-list__description', descriptionClass]"
      >
```

Is this work for you?

Reply:
As I can see we can also use here `index`, since the items cannot be manipulated (removed, add more or reorderer).

It can be like :key="`dt-${index}`"

Reply:
That worked, thanks! The term is supposed to be unique so I thinks it's fine to go with that one.